brooklyn-server/server-cli/src/main/java/org/apache/brooklyn/cli/ItemLister.java

```diff
@@ -198,17 +277,36 @@ private void mkdir(String rootDir, String dirName) {
 
         protected List<URL> getUrls() throws MalformedURLException {
             List<URL> urls = Lists.newArrayList();
-            if (jars.isEmpty()) {
-                String classpath = System.getenv("INITIAL_CLASSPATH");
-                if (Strings.isNonBlank(classpath)) {
-                    List<String> entries = Splitter.on(":").omitEmptyStrings().trimResults().splitToList(classpath);
-                    for (String entry : entries) {
-                        if (entry.endsWith(".jar") || entry.endsWith("/*")) {
-                            urls.addAll(ClassFinder.toJarUrls(entry.replace("/*", "")));
+            if (jars.isEmpty()) { //TODO: where is `jars` value write - AbstractMain.execCli(Cli<BrooklynCommand> parser, String ...args):line 250 BrooklynCommand command = parser.parse(args)
```

Comment:
It is not allowed now to parse jars and yaml at the same time. The idea was to prevent it and `jars` to have higher priority. Do you think it's better to throw exception after parsing the arguments, instead of making it impossible from the code?